pyproject.toml

```diff
@@ -1497,10 +1505,16 @@ banned-module-level-imports = ["numpy", "pandas"]
 "flask.escape".msg = "Use markupsafe.escape instead. Deprecated in Flask 2.3, removed in Flask 3.0"
 "flask.Markup".msg = "Use markupsafe.Markup instead. Deprecated in Flask 2.3, removed in Flask 3.0"
 "flask.signals_available".msg = "Signals are always available. Deprecated in Flask 2.3, removed in Flask 3.0"
+# Some specific cases
+"unittest.TestCase".msg = "Use pytest compatible classes"
 
 [tool.ruff.lint.flake8-type-checking]
 exempt-modules = ["typing", "typing_extensions"]
 
+[tool.ruff.lint.flake8-pytest-style]
+mark-parentheses = false
+fixture-parentheses = false
```

Comment:
`mark-parentheses`: if True (default for ruff) then `@pytest.mark.db_test()` is valid option for non parametrised mark otherwise `@pytest.mark.db_test`
`fixture-parentheses`: if True (default for ruff) then use `@pytest.fixture()` is valid option for non parametrised fixture otherwise `@pytest.fixture`

Particular this option generate less changes, so assumption here that it is correct selection for this rules

|                                 | **mark-parentheses  =  `false`** | **mark-parentheses  =  `true`** |
|---------------------------------|--------------------------------|-------------------------------|
| **fixture-parentheses = `false`** | 179 errors in 74 files         | 990 errors in 267 files       |
|  **fixture-parentheses = `true`** | 271 errors in 150 files        | 1082 errors in 311 files      |